src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -4,7 +4,10 @@
  * Container for user visible messages.
  */
 public class Messages {
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
+    public static final String MESSAGE_DUPLICATE_PERSON = "A person with this name already exists. "
+            + "Please do add tags to other persons with the same name to differentiate between them!";
+    public static final String MESSAGE_DUPLICATE_DETAILS = "A person with these details already exists. "
+            + "Please do add tags that differentiate between them!";
```

Comment:
`MESSAGE_DUPLICATE_PERSON` tells the user to add tags to the _other_  existing contacts with the same details, whereas `MESSAGE_DUPLICATE_DETAILS` tells the user to add tags to the _current_  contact being viewed.

I separated them in this way so that we can add the index of the duplicate contacts in the future to `MESSAGE_DUPLICATE_PERSON` when adding a person to improve usability.

Would it be better if I added the index of duplicate contacts now? Or should I just make it the same message?

Reply:
Oh I see. I think its fine then the way it is.